### ISSUE TITLE
Windows version error always on 2-pass

### ISSUE THREAD

**Issue by bitsky6** [jeanslack/Videomass#22]
I have this problem whatever settings I use with 2 pass will always give error.

Ratecontrol_init: can't open stats file

The only solution I found is to add after the video codec
-strict -2 -passlogfile /tmp/location

On Mac it is working ok without adding the extra line.

Any ideea?
Thanks

**Comment by jeanslack** [comment]
hi,

Which version of Videomass are you using? and what is the version of FFmpeg in use by Videomass?

Can you tell me the steps you took on Videomass before getting that error?
Also, log output or screenshots would be great.

**Comment by bitsky6** [comment]
Hello,

I am using the latest version of Videomass 3.4.2 and FFmpeg 4.4.4.

As for the steps whatever I select from Preset Manager that has 2-Pass or from A/V conversions if i select 2-Pass it will always give this error.

<img width="675" alt="Screenshot 2021-05-02 141254" src="https://user-images.githubusercontent.com/57507315/116813638-f7ec6780-ab54-11eb-876e-25b2dad4c363.png">
<img width="677" alt="Screenshot 2021-05-02 141332" src="https://user-images.githubusercontent.com/57507315/116813641-f91d9480-ab54-11eb-835b-a6a0b04afa9b.png">

**Comment by jeanslack** [comment]
Unfortunately I have not been able to reproduce that error and after some research I have not yet found a solution even on the web. 

What happens if you replace the code on 2-Pass with the following piece of code? 
`-c:v libx264 -b:v 1M -c:a ac3 -pass 2 -pix_fmt yuv420p -map 0:v? -map_chapters 0 -map 0:s? -map 0:a? -map_metadata 0`

**Comment by bitsky6** [comment]
It is the same.

No matter what i do if I ahve 2 pass enabled always same error.

I did try to replace my ffmpeg with another version but the same.

**Comment by bitsky6** [comment]
The only solution is to add always -strict -2 -passlogfile /tmp/mydummy to pass 1 and to pass 2.

None of the presets work with pass 2 otherwhise.

Any ideea?

thanks

**Comment by bitsky6** [comment]
Just example if I use like this in cmd ffmpeg it works the pass 2.

If I copy from videomass pass 1 and pass 2 gives the same error.

ffmpeg -y -i "input" -c:v libx264 -b:v 2600k -pass 1 -an -f null null && ffmpeg -i "input" -c:v libx264 -b:v 2600k -pass 2 -c:a aac -b:a 128k "output.mp4"

**Comment by bitsky6** [comment]
So actually i try in cmd with ffmpeg like this and it works.

I took it from the Log to see what it is doing and if I add it to ffmpeg not with Videomass it is working.

ffmpeg  -loglevel info -stats -hide_banner -nostdin  -i "input" -an -sn -c:v libx264 -b:v 1M -pass 1 -pix_fmt yuv420p -f null -threads 16 -y NUL && ffmpeg -loglevel info -stats -hide_banner -nostdin  -i "input" -c:v libx264 -b:v 1M -pass 2 -pix_fmt yuv420p -map 0:v? -map_chapters 0 -map 0:s? -map 0:a? -map_metadata 0  -threads 16 -y "output.mp4" 

But from Videomass error always...

**Comment by bitsky6** [comment]
This is what Videomass log writes:

Stream mapping:
  Stream #0:0 -> #0:0 (h264 (native) -> h264 (libx264))
[libx264 @ 0000025def55a440] using SAR=1/1
[libx264 @ 0000025def55a440] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX FMA3 BMI2 AVX2
[libx264 @ 0000026a2a917c80] ratecontrol_init: can't open stats file
Error initializing output stream 0:0 -- Error while opening encoder for output stream #0:0 - maybe incorrect parameters such as bit_rate, rate, width or height
Conversion failed!
[Videomass]: FAILED !

And this is what FFmpeg from CMD writes:

Stream mapping:
  Stream #0:0 -> #0:0 (h264 (native) -> h264 (libx264))
[libx264 @ 0000025def55a440] using SAR=1/1
[libx264 @ 0000025def55a440] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX FMA3 BMI2 AVX2
[libx264 @ 0000025def55a440] profile Main, level 4.0, 4:2:0, 8-bit
Output #0, null, to 'NUL':
  Metadata:
    encoder         : Lavf58.76.100
  Stream #0:0: Video: h264, yuv420p(tv, bt709, progressive), 1920x816 [SAR 1:1 DAR 40:17], q=2-31, 1000 kb/s, 23.98 fps, 23.98 tbn (default)
    Metadata:
      BPS-eng         : 8238300
      DURATION-eng    : 01:53:07.031000000
      NUMBER_OF_FRAMES-eng: 162726
      NUMBER_OF_BYTES-eng: 6989200287
      _STATISTICS_WRITING_APP-eng: mkvmerge v52.0.0 ('Secret For The Mad') 64-bit
      _STATISTICS_WRITING_DATE_UTC-eng: 2021-03-31 07:04:25
      _STATISTICS_TAGS-eng: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
      encoder         : Lavc58.134.100 libx264
    Side data:
      cpb: bitrate max/min/avg: 0/0/1000000 buffer size: 0 vbv_delay: N/A
frame=  236 fps=228 q=38.0 size=N/A time=00:00:07.25 bitrate=N/A speed=7.02x


There are the same commands but from Videomass it stops.

**Comment by jeanslack** [comment]
Does it happen with any video files? By the way you said that it works on Mac-OS.

**Comment by bitsky6** [comment]
Any video file is the same.

On mac Yes it works.

**Comment by jeanslack** [comment]
So I think it is the version of FFmpeg you are using that leads to having this problem on Videomass. I am using ffmpeg-4.2.2-win64-static (the one included on Videomass-v3.4.2-x86_64-portable.7z / Videomass-v3.4.2-x86_64-Setup.exe) and have never had this problem.

**Comment by bitsky6** [comment]
So I think I found the issue.

Before I go crazy. For some reason my Administrator account which is the only account has somewhere some permission issues and it is blocking videomass somewhere in the process.

What I did just to be sure, I enabled Windows local Administrator account and there it works.....

Just for future information if somebody has this kind of problems.

Thanks Anyway, good at least that I found where is the problem and it is not Videomass.

Thanks
